sparkmagic/sparkmagic/utils/constants.py

```diff
@@ -76,3 +76,11 @@
 
 LIVY_HEARTBEAT_TIMEOUT_PARAM = u"heartbeatTimeoutInSecond"
 LIVY_KIND_PARAM = u"kind"
+
+KERBEROS_KINIT = 'kinit'
+# Time in seconds
+KERBEROS_TIME_INTERVAL = 6000
```

Comment:
Also, please name it so that the user knows what time unit to use: `kerberos_renew_time_interval_seconds` or something similar.